bibliopixel/remote/trigger_process.py

```diff
@@ -0,0 +1,10 @@
+from .. util import importer
```

Comment:
I suggest you move all of this file into remote/trigger.py - they are both tiny.